source/openapi3/split.yaml

```diff
@@ -2081,6 +2081,101 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ListAllBankAccountsResponse'
+  /bank_connections:
+    get:
+      tags:
+        - BankConnections
+      summary: List all BankConnections
+      description: >
+        By default, all BankConnections will be returned. You can apply filters to your
+        query to customise the returned BankConnection list.
+      operationId: ListAllBankConnections
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ListAllBankConnectionsResponse'
+      parameters:
+        - name: page
+          in: query
+          description: 'Page of results to return, single value, exact match'
+          style: form
+          schema:
+            type: string
+          example: '1'
+        - name: per_page
+          in: query
+          description: 'Number of results per page, single value, exact match'
+          style: form
+          schema:
+            type: string
+          example: '100'
+        - name: provider_name
+          in: query
+          description: 'Single value, exact match'
+          style: simple
+          schema:
+            type: string
+            enum: [basiq, proviso, split]
+          example: basiq
+        - name: state
+          in: query
+          description: 'Single value, exact match'
+          style: simple
+          schema:
+            type: string
+            enum: [active, credentials_invalid]
+          example: credentials_invalid
+  /bank_connections/{id}:
+    get:
+      tags:
+        - BankConnections
+      summary: Get a Bank Connection
+      description: Get a single BankConnection by its ID
+      operationId: GetABankConnection
+      parameters:
+        - name: id
+          in: path
+          description: BankConnection ID (`BankConnection.data.id`)
+          required: true
+          style: simple
+          explode: false
+          schema:
+            type: string
+            format: UUID
+          example: a87f6739-1652-4438-8906-648cd11b0136
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GetABankConnectionResponse'
+        '404':
+          description: Not Found
+    delete:
+      tags:
+        - BankConnections
+      summary: Remove a BankConnection
+      description: >
+        <aside class="notice">Removing a BankConnection will not affect your
+        transaction history.</aside>
```

Comment:
I have no idea whether this ☝️ statement is correct. Can someone who knows tell me?